### ISSUE TITLE
Cube not rotating

### ISSUE THREAD

**Issue by CodeHelix** [fehaar/FFWD-Templates#6]
I was going through to test out the new version of the Logo scene with the rotating cube and I was able to get the cube show but for some reason it isn't rotating. I'm not receiving any errors so am unsure as to why the cube won't rotate as I have added the script to the scripts template. I also had to manually add all the PressPlay.FFWD dll files to all the projects in order for them to build but that's not that big of a deal.

**Comment by fehaar** [comment]
That you have to manually add the DLL's tells me that something is wrong with your setup. But if you can see the cube, you have managed to come far enough. I just tried it yesterday on all three platforms, so it is odd that it is not working.

Try two things:
1) Open the XML file for the Logo scene and search for "Spin". There should be a reference to the Spin component there. If not - reexport the scene.
2) Set a breakpoint in the Spin file and see if Update gets called.

**Comment by CodeHelix** [comment]
I thought it was weird as well as when I looked to see where the dll files were pointing to they were pointing at the right direction but for some reason it will still say the namespace was missing so had to read them back.

Also you were right about the spin component being missing which I noticed that the script wasn't attached to the cube so had to attach it and when I re-exported everything worked. Thanks for the help!
